pyseir/inference/model_fitter.py

```diff
@@ -459,23 +459,23 @@ def _fit_seir(
         l = locals()
         model_kwargs = {k: l[k] for k in self.model_fit_keys}
 
-        # Last data point used in Fit
-        last_data_point_used = t0 + t_break + 14 + t_delta_phases + 14
+        # Last data point in ramp 2
+        last_data_point_ramp_2 = t0 + t_break + 14 + t_delta_phases + 14
         # Number of future days used in second ramp period
-        number_of_future_days_used = last_data_point_used - self.days_since_ref_date
+        number_of_future_days_used = last_data_point_ramp_2 - self.days_since_ref_date
         # Multiplicative chi2 penalty if future_days are used in second ramp period (set to 1 by default)
         future_days_penalty = 1.0
 
         # If using more future days than allowed, updated future_days_penalty
         if number_of_future_days_used > self.future_days_allowed:
             future_days_penalty = number_of_future_days_used
 
-        # Only run fit when last_data_point_used does not use more than max_future_days_fitted
-        if last_data_point_used < self.days_since_ref_date + self.max_future_days_fitted:
-            model = self.run_model(**model_kwargs)
+        # Only run fit when last_data_point_ramp_2 does not use more than max_future_days_fitted
+        # if last_data_point_ramp_2 < self.days_since_ref_date + self.max_future_days_fitted:
+        model = self.run_model(**model_kwargs)
         # Otherwise return chi2 = 1000, we could further optimize this, but this is functional
-        else:
-            return 1000
+        # else:
```

Comment:
Delete this commented code and the above commented code. Git records your changes, so you can always look at historical editions.